src/msg/async/rdma/Infiniband.cc

```diff
@@ -1043,7 +1043,7 @@ Infiniband::CompletionQueue* Infiniband::create_comp_queue(
 
 // 1 means no valid buffer read, 0 means got enough buffer
 // else return < 0 means error
-int Infiniband::recv_msg(CephContext *cct, int sd, IBSYNMsg& im)
+int Infiniband::recv_msg(CephContext *cct, int sd, ib_cm_meta& im)
```

Comment:
rename patch title to be "msg/async/rdma: rename type name IBSYNMsg to ib_cm_meta"

Reply:
Thanks, got it.

Reply:
@iotcg could you avoid "resolving" the comment? otherwise i have to check every single one of them when i start a new round of review even if some of them have been resolved in previous review.

Reply:
Sorry for the mistake.